datasync/chaindatafetcher/kafka/consumer.go

```diff
@@ -18,17 +18,74 @@ package kafka
 
 import (
 	"context"
+	"encoding/binary"
 	"errors"
 	"fmt"
 
 	"github.com/Shopify/sarama"
 )
 
-var eventNameErrorMsg = "the event name must be either 'blockgroup' or 'tracegroup'"
+//go:generate mockgen -destination=./mocks/consumer_group_session_mock.go -package=mocks github.com/klaytn/klaytn/datasync/chaindatafetcher/kafka ConsumerGroupSession
+// ConsumerGroupSession is for mocking sarama.ConsumerGroupSession for better testing.
+type ConsumerGroupSession interface {
```

Comment:
Are you planning to make more test regarding to this interface?
I don't see why you made this. The code works fine without a interface

Reply:
This interface is used for the unit-testing. In `TestConsumer_updateOffset` test, there is a mock for the interface.